schrack/integrationpluginschrack.cpp

```diff
@@ -124,19 +124,26 @@ void IntegrationPluginSchrack::setupThing(ThingSetupInfo *info)
     // The maxChargingCurrentE3 takes into account the DIP switches and connected cable, so this is effectively
     // our maximum. However, it will go to 0 when unplugged, which is odd, so we'll ignore 0 values.
     connect(cionConnection, &CionModbusRtuConnection::maxChargingCurrentE3Changed, thing, [=](quint16 maxChargingCurrentE3){
-        qCDebug(dcSchrack()) << "Maximum charging current E3 current changed:" << maxChargingCurrentE3;
+        qCInfo(dcSchrack()) << "Maximum charging current E3 current changed:" << maxChargingCurrentE3;
         if (maxChargingCurrentE3 != 0) {
             thing->setStateMaxValue(cionMaxChargingCurrentStateTypeId, maxChargingCurrentE3);
         }
     });
 
-    connect(cionConnection, &CionModbusRtuConnection::statusBitsChanged, thing, [=](quint16 /*statusBits*/){
+    connect(cionConnection, &CionModbusRtuConnection::statusBitsChanged, thing, [=](quint16 statusBits){
         thing->setStateValue(cionConnectedStateTypeId, true);
-//        qCDebug(dcSchrack()) << "Status bits changed:" << statusBits;
+        StatusBits status = static_cast<StatusBits>(statusBits);
+        // TODO: Verify if the statusBit for PluggedIn is reliable and if so, use that instead of the plugged in time for the plugged in state.
+        qCInfo(dcSchrack()) << "Status bits changed:" << status;
     });
 
     connect(cionConnection, &CionModbusRtuConnection::minChargingCurrentChanged, thing, [=](quint16 minChargingCurrent){
-        qCDebug(dcSchrack()) << "Minimum charging current changed:" << minChargingCurrent;
+        qCInfo(dcSchrack()) << "Minimum charging current changed:" << minChargingCurrent;
```

Comment:
I personally would not inform on every state changed with `qInfo` since info is always enabled by default. Maybe thing setup and stuff is interesting to see from a plugin by default or discovery started/finished, but not states.

Reply:
Yeah.. I think the idea way that debug prints the whole register set on every update, which floods the log so much that one would miss those changed signals...

But you're right, as long as we've enabled Info by default, this isn't good.

That said, I propose to reduce the default log level to Warning, except maybe for dcApplication() so it still prints the version info at startup.